docs/content/quickstart/installation.md

```diff
@@ -7,60 +7,60 @@ chapter = false
 
 ## Introduction
 
-The Kairos releases ships for user convenience a set of artifacts that can be used to install kairos on a node. 
-However, Kairos Kubernetes Native components allows the creation of these artifacts inside Kubernetes from a set of input images. 
+The Kairos releases ships for user convenience a set of artifacts that can be used to install Kairos on a node. 
+However, Kairos Kubernetes Native components allow the creation of these artifacts inside Kubernetes from a set of input images. 
 
-In the quickstart below we will use the artifacts generated by the kairos releases as we assume we don't have any prior Kubernetes cluster available.
+In the quickstart below, we will use the artifacts generated by the Kairos releases, as we assume we don't have any prior Kubernetes cluster(s) available.
```

Comment:
Either we use quickstart or quick start, but everywhere. Here quick start is used in the title, then quickstart is used in this sentence